android/cli/commands/_build.js

```diff
@@ -35,8 +35,8 @@ var ADB = require('node-titanium-sdk/lib/adb'),
 	temp = require('temp'),
 	SymbolLoader = require('appc-aar-tools').SymbolLoader,
 	SymbolWriter = require('appc-aar-tools').SymbolWriter,
-	ti = require('titanium-sdk'),
-	tiappxml = require('titanium-sdk/lib/tiappxml'),
+	ti = require('node-titanium-sdk'),
+	tiappxml = require('node-titanium-sdk/lib/tiappxml'),
```

Comment:
@janvennemann Can you do that? I'll be out until Wednesday and we should really get that in before we forget it. Maybe in an own PR which we can merge before this one gets queued. Thanks guys!